pom.xml

```diff
@@ -11,7 +11,7 @@
 	<groupId>io.jenkins.services.distribution-service</groupId>
 	<artifactId>custom-distribution-service</artifactId>
 	<version>0.0.1</version>
-	<name>Custom Jenkins Distribution Service</name>
+	<name>custom jenkins distribution service</name>
```

Comment:
I'm not sure why you do the changes here.

Reply:
Yes I am changing this back, just a minor correcting while updating package names